terratorch/tasks/wxc_downscaling_task.py

```diff
@@ -42,9 +44,9 @@ def configure_models(self) -> None:
         self.model: Model = self.model_factory.build_model(
             "regression", aux_decoders=None, model_config=self.model_config, **self.hparams["model_args"]
         )
-        if self.hparams["freeze_backbone"]:
+        if self.hparams_["freeze_backbone"]:
```

Comment:
@Joao-L-S-Almeida what's the difference between hparams and hparams_?

Reply:
@CarlosGomes98 cc maybe we should come up with a better name for hparams_ which tells us the difference

Reply:
I defined `self.hparams_` to be a dictionary containing all the original model config, since `self.hparams` (which I understand is being magically created by Lightning behind the scenes), is incomplete for my purposes. 
I renamed it to `self.extended_hparams` in commit [384eae7](https://github.com/IBM/terratorch/pull/179/commits/384eae789cea0e8e9f4db4ae6e0199dd8365e0ae).